_data/impls/absolutetelnet.yml

```diff
@@ -0,0 +1,51 @@
+name: AbsoluteTelnet
+homepage: www.celestialsoftware.com
```

Comment:
This should be a full URL, including protocol, i.e. add "http://". Also, I can't find anything about AbsoluteTelnet on that page. After searching, it seems you may have meant http://www.celestialsoftware.net/ ?